Sources/NIOTransportServices/Datagram/NIOTSDatagramConnectionBootstrap.swift

```diff
@@ -17,9 +17,13 @@ import NIOCore
 import Dispatch
 import Network
 
-/// A `NIOTSDatagramBootstrap` is an easy way to bootstrap a `NIOTSDatagramChannel` when creating network clients.
+@available(OSX 10.14, iOS 12.0, tvOS 12.0, watchOS 6.0, *)
+@available(*, deprecated, renamed: "NIOTSDatagramConnectionBootstrap")
+public typealias NIOTSDatagramBootstrap = NIOTSDatagramConnectionBootstrap
```

Comment:
I understand if we may want to avoid this - I can roll this change back if we don't want to deprecate the existing type.

Reply:
Sure, if we're okay with the type alias then I'd vote for that but without deprecating.